### PR TITLE
US-81 | Remove Linked Events dependency from location import

### PR DESCRIPTION
**Pull request by tuomas777**
Before this change, every imported location was enriched with place data from Linked Events. The enriched data wasn't used for much, and doing the enrichment caused a lot of request to LE making the location import slow and less robust, so it made sense to get rid of that.

The only thing that was actually used from the enriched data were administrative divisions. For those, there is now a separate importer which uses [django-munigeo](https://github.com/City-of-Helsinki/django-munigeo) for fetching the divisions. Because munigeo requires a database and US isn't actually using one ATM, a temporary Spatialite database is used just for storing the divisions for the duration of the imports.

**!!! NOTE !!!** Administrative division importing is broken in [the latest munigeo (v0.3.6)](https://github.com/City-of-Helsinki/django-munigeo/releases/tag/v0.3.6), so this PR uses a custom fork where the issues have been fixed. We might want to wait for those fixes to land in munigeo upstream before merging this PR.

### CODE REVIEW

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -151,3 +151,17 @@ jobs:
...
           command: "{/bin/sh}"
           args: "{-c,python manage.py ingest_data ontology_tree; python manage.py ingest_data ontology_word}"
+
+      - name: Harvest administrative division indexes
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          name: harvest-administrative-division-indexes
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}/sources
+          image_tag:  ${{ github.sha }}
+          secret_name: project-staging-sources-secret
```
Comment (nikomakela) [inline .github/workflows/production.yml:161]: Is staging good for production? I have understood that we don't have a production server yet, but should we start using production secrets etc so we don't miss them when the production server will be built?

File: .github/workflows/production.yml [hunk 2]
```diff
@@ -151,3 +151,17 @@ jobs:
...
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW}}
+          target_namespace: ${{ secrets.K8S_NAMESPACE_STAGING }}
```
Comment (nikomakela) [inline .github/workflows/production.yml:163]: "staging"

**Review by nikomakela — COMMENTED** [review]
I ran `docker-compose exec sources python manage.py ingest_data location
` and it went through without issues. It fetched 17920 item in total from `hauki.api.hel.fi` and it took less than 554 sec.

Code changes looked good to me, but I would start using production variables in prod configuration. **The django-munigeo needs an official release!** - When it's available, the changes are good to go.

Is there any data that will be missing after this change from LinkedEvents to Munigeo?

File: .github/workflows/production.yml (hunk 2 above)
Comment (tuomas777) [reply .github/workflows/production.yml]: This and the above one were caused by a copy-paste mistake, good catch! 💯

**Comment by tuomas777** [comment]
> Code changes looked good to me, but I would start using production variables in prod configuration. **The django-munigeo needs an official release!** - When it's available, the changes are good to go.

There is no actual release yet, but the needed changes are now merged into django-munigeo upstream master, so I think we can install from the commit there for now.

> Is there any data that will be missing after this change from LinkedEvents to Munigeo?

No. Linked Events itself uses munigeo to create the divisions.

**Pull request merged**